src/pages/[platform]/build-a-backend/add-aws-services/rest-api/test-api/index.mdx

```diff
@@ -0,0 +1,152 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+                  
+export const meta = {
+  title: 'Test the REST API',
+  description: 'Learn how you can test the REST API from the terminal, with Amplify Mock, or with the API Gateway console.',
+  platforms: [
+    'android',
+    'angular',
+    'flutter',
+    'javascript',
+    'nextjs',
+    'react',
+    'react-native',
+    'swift',
+    'vue'
+  ],
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+          
+
+## Test the API from the terminal
+
+If Guest users have access to your REST API you can test it from the terminal using curl.
+
+[Curl](https://github.com/curl/curl) is a command-line tool that lets you transfer data to and from a server using various protocols.
+
+> Curl is available in many distributions including Mac, Windows and Linux. Follow the install instructions in the [docs](https://curl.haxx.se/docs/install.html).
+
+<BlockSwitcher>
+<Block name="Mac and Linux">
+
+### GET method example
+
+```bash title="Terminal" showLineNumbers={false} 
+curl https://a5b4c3d2e1.execute-api.eu-west-2.amazonaws.com/dev/items
+```
+
+### POST method example
+
+```bash title="Terminal" showLineNumbers={false} 
+curl -H "Content-Type: application/json" -d '{"name":"item-1"}' https://a5b4c3d2e1.execute-api.eu-west-2.amazonaws.com/dev/items
+```
+
+</Block>
+<Block name="Windows">
+
+### GET method example
+
+```bash title="Terminal" showLineNumbers={false} 
+curl https://a5b4c3d2e1.execute-api.eu-west-2.amazonaws.com/dev/items
+```
+
+### POST method example
+
+```bash title="Terminal" showLineNumbers={false} 
+curl -H "Content-Type: application/json" -d {\"name\":\"item-1\"} https://a5b4c3d2e1.execute-api.eu-west-2.amazonaws.com/dev/items
+```
+
+</Block>
+</BlockSwitcher>
+
+<Callout warning="true">
+
+**Warning:** Testing methods using production endpoints may result in changes to resources that cannot be undone.
```

Comment:
what kind of changes result from testing the methods? or is this referring to unintentionally setting data in your prod env?

Reply:
the warning is geared to towards the data modification the call might make